app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        command, source, destination = command.split()
+    except ValueError:
+        print("Three arguments expected")
+        return
+    if command != "mv" or source == destination:
+        return
+    if not os.path.exists(source):
+        raise FileNotFoundError("Source file does not exist")
+
+    if "/" not in destination:
+        os.rename(source, destination)
+        return
+
+    directories = destination.split("/")[:-1]
+    path = ""
```

Comment:
you can use os.path.split
Usage of .split("/") is not correct because on different os it will works differently